tooling/pack-macros.mjs

```diff
@@ -1,35 +1,49 @@
 /**
- * Pack previously-unpacked JSON/JS files into a Foundry `.db`
+ * Pack previously-unpacked JSON/JS files into a Foundry LDB.
```

Comment:
Reworked to pack from a folder structure into a v11+ DB. No other functional changes.